packages/i18n/locales/cs/translation.json

```diff
@@ -0,0 +1,629 @@
+{
+    "button": {
+      "acceptTerms": "Přijmout podmínky",
+      "add": "Přidat",
+      "addAnAction": "Přidat akci",
+      "addCollection": "Přidat kolekci",
+      "addMember": "Přidat člena",
+      "addMembers": "Přidat členy",
+      "addPayment": "Vytvořit platbu",
+      "addTier": "Přidat hodnost",
+      "addToKeplr": "Přidat token do Keplru",
+      "castYourVote": "Hlasovat",
+      "claim": "Vybrat",
+      "claimNumTokens": "Vybrat {{amount}} ${{tokenSymbol}}",
+      "clickHereToProposeAdding": "Pro přidání klikněte zde.",
+      "clickToCopyAddress": "Zkopírovat adresu",
+      "close": "Zavřít",
+      "connectWallet": "Připojit peněženku",
+      "continue": "Pokračovat",
+      "copy": "Kopírovat",
+      "copyAddressToClipboard": "Zkopírujte adresu",
+      "copyDaoAddress": "Zkopírujte DAO adresu",
+      "copyToClipboard": "Zkopírujte",
+      "create": "Vytvořit",
+      "createAProposal": "Vytvořit návrh",
+      "createAToken": "Vytvořit token",
+      "createDAO": "Vytvořit DAO",
+      "createSubDao": "Vytvořit SubDAO",
+      "deleteDraft": "Smazat návrh",
+      "deposit": "Vklad",
+      "deselectAllNfts": "Zrušte výběr všech {{count}} NFTs",
+      "done": "Hotovo",
+      "duplicate": "Zduplikovat",
+      "execute": "Provést",
+      "follow": "Sledovat",
+      "following": "Sledujete",
+      "goBack": "Zpět",
+      "goToDaoPage": "DAO stránka",
+      "gotIt": "Rozumím",
+      "hidePreview": "Skrýt náhled",
+      "hideRawData": "Skrýt nezpracovaná data",
+      "import": "Importovat",
+      "installKeplr": "Stáhnout peněženku",
+      "loadDraft": "Načíst návrh",
+      "loadMore": "Načíst více",
+      "manage": "Správa",
+      "manageStake": "Pravovat ${{tokenSymbol}} vklad",
+      "newProposal": "Nový návrh",
+      "newSubDao": "Nové SubDAO",
+      "numProposalsInInbox_one": "Máte {{count}} nespracovaný návrh",
+      "numProposalsInInbox_other": "Máte {{count}} nespracovaných návrhů",
+      "numProposalsInInbox_zero": "Nemáte nespracované návrhy",
+      "openInDestination": "Otevřít v {{destination}}",
+      "preview": "Náhled",
+      "proposeUpgrade": "Navrhnout upgrade",
+      "publish": "Publikovat",
+      "resetQuestion": "Reset?",
+      "review": "Kontrola",
+      "save": "Uložit",
+      "saveDraft": "Uložit návrh",
+      "searchDaos": "Najít DAO",
+      "selectAllNfts": "Označit všechna {{count}} NFTs",
+      "setDisplayName": "Nastavit zobrazované jméno",
+      "showInstantiateMessage": "Zobrazit zprávu",
+      "showRawData": "Zobrazit zdrojová data",
+      "stake": "Uzamknout",
+      "stakeAllButProposalDeposit": "Zamknout vše kromě {{proposalDeposit}} ${{tokenSymbol}} vkladu návrhu",
+      "stakeOrUnstake": "Uzamknout nebo odemknout",
+      "stakeTokenSymbol": "Zamknout ${{tokenSymbol}}",
```

Comment:
když už někde používáme uzamknout, tak bych to nechal všude.

Reply:
FIxed